script.js

```diff
@@ -0,0 +1,180 @@
+const moreElements = function () {
```

Comment:
- [ ] You have done a good job with your capstone :raised_hands: but one of the project requirements was to generate the speakers' section dynamically with JavaScript. I would suggest storing all the speakers' information in a variable and then using it to create the speakers' section in order to match the project requirements. Please, kindly address this issue :pushpin: :smiley:
![image](https://user-images.githubusercontent.com/81522435/146794124-222af812-4b6a-4d9b-9d19-8512b18ee1a6.png)